content/news/2016/03/2016-03-08-using-code-to-spur-innovation.md

```diff
@@ -7,10 +7,8 @@ authors:
   - andreanocesigritz
 topics:
   - code
```

Comment:
```suggestion
  - software-engineering
```

Reply:
Use: innovation, crowdsourcing-and-citizen-science, mobile, apps-challenges